src/multiply/download/genomes.py

```diff
@@ -100,6 +100,53 @@ def create_genome(self, name, genus, species, strain, include_variation=None):
 
         return genome
 
+class VectorBaseFactory(GenomeFactory):
+    """
+    Create Genome objects from VectorBase
+
+    """
+
+    source = "vectorbase"
+    source_url = "https://vectorbase.org/common/downloads"
+
+    def create_genome(self, name, genus, species, strain, release, include_variation=None):
```

Comment:
For the other GenomeFactory subclasses, I made "release" a class attribute. But I think this approach is nice as it gives user some more control. Perhaps best would be to have a sensible default to keep each new genome as easy to define as possible, while still giving control for those who want it. Whatever final decision, probably best to have consistency across all the subclasses.